### ISSUE TITLE
The index page charts have still issues - demo-dev

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#1335]
The descendants, hourglass and the pedigree chart blocks are now too high with the vertical scrollbars.

The descendants chart has also the following issues
1. it does not yet have a bottom scroll bar, so we do not see the left hand of the chart. 
2. there is a - line to the right of the root person's box.

See the original issue https://github.com/fisharebest/webtrees/issues/1324#issuecomment-324712109

![image](https://user-images.githubusercontent.com/5202996/29741616-6b7e6c94-8a79-11e7-8125-bf93029afd66.png)

**Comment by melizaa** [comment]
The issues exist in all the themes.

**Comment by JustCarmen** [comment]
> it does not yet have a bottom scroll bar, so we do not see the left hand of the chart.

@makitso I think this can be solved by using flexbox on the outside container:
```
.wt-block-content-charts-block {
    display: flex;
    flex-flow: row-reverse;
}
```

**Comment by makitso** [comment]
This does not appear to work in Firefox 55.0.2

**Comment by JustCarmen** [comment]
Sorry, didn't tested it in those browsers. I'm using Chrome.

**Comment by makitso** [comment]
Posted a fix to #1 above.the bad news is that the chart starts left justified not right.  IMO, a acceptable compromise.
